website/views/options/social/challenges.jade

```diff
@@ -128,12 +128,25 @@ script(type='text/ng-template', id='partials/options.social.challenges.html')
               .form-group
                 textarea.form-control(cols='3', placeholder=env.t('challengeDescr'), ng-model='newChallenge.description')
 
+<<<<<<< HEAD
               .form-group
                 .input-group
                   span.input-group-addon
                     .Pet_Currency_Gem1x
                   input.form-control(type='number', min="{{newChallenge.group=='habitrpg' ? 1 : 0}}", max="{{maxPrize}}", ng-model='newChallenge.prize', placeholder=env.t('prize'))
                 a.hint(popover="{{newChallenge.group=='habitrpg' ? env.t('prizePopTavern') : env.t('prizePop')}}", popover-trigger='mouseenter', popover-placement='right')
+=======
```

Comment:
Hmm. That's weird. I'm not sure how this got passed when I use --continue, but I was having some trouble